standardC.css

```diff
@@ -0,0 +1,74 @@
+
+.container {
+    text-align: inline;
+}
+
+/* Standard | Scienctific */
+
+p {
+    font-size: 40px;
+    margin-left: 35px;
+    padding-left: 15px;
+    padding-top: 15px;
+    color: rgb(249, 249, 252);
+  }
+
+.card{ 
+   background-color: gray; /*(87, 23, 23) */
+   margin-right: auto;
+   margin-left: auto;
+   border-radius: 5%;
+   width: 10;
+   
+}
+
+/* Numerical Input */
+
+.input { 
+    text-align: right;
+    font-size: 50px;
+    margin-left: 45px;
+    margin-right: 45px;
+    
+    display: inline-block;
+    border-radius: 3%;
+    width: 665px;
+    height: 70px;  
+}
+
+.button:hover{
+    background-color: blue;
+    transition-duration: 0.4s;
+}
+
+/* All Buttons */
+
+.button{
+    margin: 5px 15px 5px 5px;
+    font-size: 30px;
+    margin-left: 10px;
+    margin-right: 10px;
+    width: 150px;
+    padding: 30px;
+    border-radius: 10%;
+    background-color: rgb(143, 55, 29);
+    color: rgb(249, 249, 252);
+ 
+}
+   
+.numeric-button {
```

Comment:
Right idea but you applied this styling to all non numeric buttons. Perhaps naming it non-numeric-button?